src/vaadin-combo-box-mixin.html

```diff
@@ -685,9 +685,12 @@
           this.value = '';
         }
       } else {
+        const itemsMatchedByLabel = this.filteredItems
```

Comment:
Didn't understand it from the ticket specifications. Should we have one **exact** match or should it be **one** match? For example, "Finlan" -> "Finland".

Reply:
My understating is that is should be **exact**.  Setting on partial matching would be a new feature. See https://github.com/vaadin/vaadin-combo-box/issues/775#issuecomment-501979966